terracumber_config/tf_files/salt-shaker/Salt-Shaker-Saltstack-Leap15.4.tf

```
@@ -0,0 +1,113 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = string
+  default = "https://ci.suse.de/user/manager/my-views/view/Salt%20Shaker/job/manager-salt-shaker-saltstack-leap154"
+}
+
+// Not really used as this is for --runall parameter, and we run cucumber step by step
+variable "CUCUMBER_COMMAND" {
+  type = string
+  default = "echo EXECUTE SALT TESTS HERE"
+}
+variable "CUCUMBER_BRANCH" {
+  type = string
+  default = "master"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = string
+  default = "/root/"
+}
+
+variable "MAIL_SUBJECT" {
+  type = string
+  default = "Results Salt Shaker - saltstack - LEAP 15.5 $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
```

The version is wrong here. I would also suggest a change:
```suggestion
  default = "Results Salt Shaker - saltstack - openSUSE Leap15.4 $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
```